### DISCUSSION TITLE
Requesting old v4 documentation

### DISCUSSION THREAD

**Discussion by aavelyn** [vmware/clarity#5583] [Q&A] [answered]
Is the old v4 documentation (app) still available somewhere? 
We haven't upgrade to v5 yet and still need access to old code examples and snippets.

**Comment by bbogdanov** [comment, marked as answer]
Hi, you can find the v4 documentation at https://v4.clarity.design/

Also, in case you would to see even older ones you can find them at https://clarity.design/releases/

**Reply by aavelyn** [reply]
Ah ok thanks a lot!
